prql-java/prql-jdbc/src/main/java/org/prqllang/jdbc/CustomClassLoader.java

```diff
@@ -0,0 +1,119 @@
+package org.prqllang.jdbc;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Files;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+final class CustomClassLoader extends URLClassLoader {
```

Comment:
Is this file from somewhere or was it written for PRQL specifically? We should link to the source if the former (and for any other files)

Reply:
Good catch. It was copied from an open source project I created years ago, with minor changes to recognize `~/` as user home directory. I guess I don't have to put reference link in this case?

I did copy code from other project - please comment on [this one](https://github.com/PRQL/prql/pull/1929/files#r1115122005).